mythril/laser/smt/bitvec.py

```diff
@@ -176,6 +522,12 @@ def If(a: Union[Bool, bool], b: Union[BitVec, int], c: Union[BitVec, int]) -> Bi
     :param c:
     :return:
     """
+    # TODO: Handle BitVecFunc
+    if isinstance(b, BitVecFunc):
+        die()
```

Comment:
What is this function? I didn't see it's definition anywhere

Reply:
whoops. XD

Reply:
Using an assert might be good in such circumstances